src/store/reducers.js

```diff
@@ -109,10 +109,12 @@ function onSetFilter(state, { payload }) {
 }
 
 function onSetPagination(state, { payload }) {
+    const perPage = parseInt(payload.perPage, 10);
+    const page = parseInt(payload.page, 10);
     return {
         ...state,
-        perPage: payload.perPage,
-        page: payload.page
+        perPage: isNaN(perPage) ? perPage : 50,
```

Comment:
should not it be reversed? `isNaN(perPage) ? 50 : perPage,`

Reply:
Yeah, my bad :facepalm: